demos/demo_simulate_solo8.py

```diff
@@ -17,11 +17,17 @@
 
 if __name__ == "__main__":
 
-    # Create a Pybullet simulation environment
+# ! Create a Pybullet simulation environment before any robots !
     env = BulletEnvWithGround()
 
-    # Create a robot instance. This initializes the simulator as well.
-    robot = env.add_robot(Solo8Robot)
+    # Create a robot instance. This adds the robot to the simulator as well.
+    robot = Solo8Robot(useFixedBase=False)
```

Comment:
Similar as above, I think the `useFixedBase = False` is the default?

Reply:
ok yeah, makes sense